core-bundle/assets/scripts/core.js

```diff
@@ -1668,7 +1668,7 @@ window.Backend =
 				}
 			},
 			clickEvent = function(e) {
-				if (e.target instanceof HTMLImageElement || e.target instanceof HTMLAnchorElement || e.target instanceof HTMLButtonElement) {
+				if (e.target instanceof HTMLAnchorElement || e.target instanceof HTMLButtonElement || e.target instanceof HTMLInputElement || e.target?.closest('a, button, input')) {
```

Comment:
Why did you change this? It is not necessary to check the parent element here.

Reply:
why would it not be necessary here? If you click on a DCA list item to enable the checkbox, you don't want to fire this action. I also added additional menu handling in 7e93013f483e243e4eb0726b4acf7832cae3050b, best explained in this video:


https://github.com/user-attachments/assets/c1c9dd55-412d-4be7-b813-c84a8ee2d5bb